test_ssh_class failure on master

Runs on Travis are currently failing:

```
________________________________ test_ssh_class ________________________________

setup_ssh = {'container_id': '6b688eeeb1f14e76e4b6b9c93cf86f4e23eb22f6c719c60865c27c883071456a', 'custom': {'host': 'localhost', 'password': 'root', 'port': 49000, 'user': 'root'}, 'port': 49000}
resource_test_dir = '/home/travis/.cache/niceman/resource_test'

    def test_ssh_class(setup_ssh, resource_test_dir):
        with swallow_logs(new_level=logging.DEBUG) as log:
    
            # Test connecting to test SSH server.
            # TODO: Add a test using a SSH key pair.
            config = dict(
                name='ssh-test-resource',
                type='ssh',
                **setup_ssh['custom']
            )
            resource = ResourceManager.factory(config)
            updated_config = resource.create()
            config.update(updated_config)
            assert re.match('\w{8}-\w{4}-\w{4}-\w{4}-\w{12}$',
                            resource.id) is not None
            assert resource.status == 'N/A'
    
            # Test running commands in a resource.
            resource.connect()
            command = ['apt-get', 'install', 'bc']
            resource.add_command(command)
            command = ['ls', '/']
            resource.add_command(command)
>           resource.execute_command_buffer()

niceman/resource/tests/test_ssh.py:71: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
niceman/resource/base.py:332: in execute_command_buffer
    session.execute_command(command['command'], env=command['env'])
niceman/resource/session.py:195: in execute_command
    **run_kw
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = SSHSession(ssh=<paramiko.client.SSHClient object at 0x7f05af259fd0>)
command = ['apt-get', 'install', 'bc'], env = None, cwd = None

    @borrowdoc(Session)
    def _execute_command(self, command, env=None, cwd=None):
        # TODO -- command_env is not used etc...
        # command_env = self.get_updated_env(env)
        if env:
            raise NotImplementedError("passing env variables to execution")
    
        if cwd:
            raise NotImplementedError("implement cwd support")
        # if command_env:
            # TODO: might not work - not tested it
            # command = ['export %s=%s;' % k for k in command_env.items()] + command
    
        escaped_command = ' '.join(quote(s) for s in command)
        stdin, stdout, stderr = self.ssh.exec_command(escaped_command)
        exit_code = stdout.channel.recv_exit_status()
        stdout = utils.to_unicode(stdout.read(), "utf-8")
        stderr = utils.to_unicode(stderr.read(), "utf-8")
    
        if exit_code not in [0, None]:
            msg = "Failed to run %r. Exit code=%d. out=%s err=%s" \
                % (command, exit_code, stdout, stderr)
>           raise CommandError(str(command), msg, exit_code, stdout, stderr)
E           niceman.support.exceptions.CommandError: CommandError: command '['apt-get', 'install', 'bc']' failed with exitcode 100.
E           Failed to run ['apt-get', 'install', 'bc']. Exit code=100. out=Reading package lists...
E           Building dependency tree...
E           Reading state information...
E            err=E: Unable to locate package bc

niceman/resource/ssh.py:147: CommandError
------------------------------ Captured log call -------------------------------
base.py                    331 DEBUG    Running command '['apt-get', 'install', 'bc']'

```

--- a/niceman/resource/tests/test_ssh.py
+++ b/niceman/resource/tests/test_ssh.py
@@ -64,6 +64,8 @@ def test_ssh_class(setup_ssh, resource_test_dir):
 
         # Test running commands in a resource.
         resource.connect()
+        command = ['apt-get', 'update']
+        resource.add_command(command)
         command = ['apt-get', 'install', 'bc']
         resource.add_command(command)
         command = ['ls', '/']

